rmgpy/rmg/model.py

```diff
@@ -130,11 +130,11 @@ def merge(self, other):
         finalModel.species.extend(uniqueSpecies)
     
         # Renumber the unique species (to avoid name conflicts on save)
-        speciesIndex = 0
-        for spec in finalModel.species:
-            if spec.label not in ['Ar','N2','Ne','He']:
-                spec.index = speciesIndex + 1
-                speciesIndex += 1
+        #speciesIndex = 0
+        #for spec in finalModel.species:
+        #    if spec.label not in ['Ar','N2','Ne','He']:
+        #        spec.index = speciesIndex + 1
+        #        speciesIndex += 1
```

Comment:
Since you implemented a different indexing system, I would just remove this section instead of commenting it out.

Reply:
will do.